chain/network/src/common/message_wrapper.rs

```diff
@@ -0,0 +1,61 @@
+use actix::dev::MessageResponse;
+use actix::Message;
+use near_rate_limiter::ThrottleController;
+use std::marker::PhantomData;
+
+// Wrapper around Actix messages, used to track size of all messages sent to PeerManager.
+// TODO(#5155) Finish implementation of this.
+
+#[allow(unused)]
+/// TODO - Once we start using this `ActixMessageWrapper` we will need to make following changes
+/// to get this struct to work
+/// - Add needed decorators. Probably `Debug`, `Message` from Actix, etc.
+/// - Add two rate limiters (local per peer, global one)
+/// - Any other metadata we need debugging, etc.
+pub struct ActixMessageWrapper<T, Q> {
+    msg: T,
+    throttle_controller: ThrottleController,
+    msg_len: usize,
+    ph: PhantomData<Q>,
+}
+
+impl<T, Q> ActixMessageWrapper<T, Q> {
+    #[allow(unused)]
+    pub fn new(msg: T, rate_limiter: ThrottleController, msg_len: usize) -> Self {
```

Comment:
`rate_limiter: ThrottleController`

Let's use consistent terminology: I think this' relatively important that near_xxx crate, XXXFrameReader, XXController and `xxx: XXXContrloller` all use the same known for `xxx`.

Reply:
Good idea, I renamed package to `near-throttled`